pkg/api/api.proto

```diff
@@ -253,6 +253,13 @@ message UpstreamNotDeployed {
 message Environment {
 
   message Application {
+    message Environment {
+      message DeploymentMetaData {
+        string deployAuthor = 1;
```

Comment:
Shouldn't these fields be in the Environment_Application?

Reply:
ahh yes, I think that was my suggestion. But now I see the redundancy. I also think it should go in the `Environment_Application`.